src/main/webapp/resources/js/pages/projects/samples-metadata-import2/components/SampleMetadataImportSteps.jsx

```diff
@@ -11,12 +11,12 @@ const { Step } = Steps;
  * @constructor
  */
 export function SampleMetadataImportSteps({ currentStep }) {
-    return (
-        <Steps current={currentStep}>
-            <Step title="Upload File" />
-            <Step title="Map Headers" />
-            <Step title="Review Data" />
-            <Step title="Complete" />
-        </Steps>
-    );
+  return (
+    <Steps current={currentStep} >
+      <Step title="Upload File" />
```

Comment:
Don't forget to internationalize theses

Reply:
Please see 90af8b23a1eff684741c80f9de9255c4b945eb20.